heft-plugins/heft-dev-cert-plugin/heft-plugin.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/heft/heft-plugin.schema.json",
+
+  "lifecyclePlugins": [],
+
+  "taskPlugins": [
+    {
+      "pluginName": "ConfigureWebpackDevServerPlugin",
```

Comment:
Yes they are intended to be unique. We explicitly block using multiple plugins with the same name but different resolved module paths (we allow them if they have the same resolved module path, since it wouldn't pose any concerns for CLI parameter names due to referencing the same set of parameters no matter what).

We could drop the unique name requirement for task plugins if we didn't care about CLI parameter collisions... but I figured it'd be best to stop those before they happen and confuse someone. The likelihood of that happening vs. someone wanting to use two separate versions of the same plugin resolved to different paths seemed like a fair tradeoff.